### DISCUSSION TITLE
The searchbar suggest list blocked

### DISCUSSION THREAD

**Discussion by chenmnkken** [community/community#29743] [Other Feature Feedback, Questions, & Ideas]
In Safari version 15.6, the GitHub homepage searchbar list blocked, hope to fix it soon, just look the below screenshot.

<img width="371" alt="Screen Shot 2022-08-17 at 10 15 49" src="https://user-images.githubusercontent.com/1049101/185020497-6d70fbba-2550-43e2-9d62-d4b3a4466575.png">

**Comment by radar-blip** [comment]
Someone else is having this same issue: #29739

You could report is as a bug if you want, here: [Bug-Report](https://support.github.com/contact/bug-report)
